main.py

```diff
@@ -284,10 +308,20 @@ def render_field_link(field):
 
 
 def render_scouting_observation_link(scouting_observation):
-    id = scouting_observation['id']
-    return '<a href="{link}">{id}</a>'.format(
-        link=url_for('scouting_observation', scouting_observation_id=id),
-        id=id)
+    oid = scouting_observation['id']
+    return '<a href="{link}">{oid}</a>'.format(
+        link=url_for('scouting_observation', scouting_observation_id=oid),
+        oid=oid)
+
+
+def render_attachment_link(scouting_observation_id, attachment):
+    aid = attachment['id']
+    return '{aid}:<a href="{link}" >Get contents</a>'.format(
```

Comment:
can we add a little bit of whitespace between attachment ID and "Get contents" link?

Reply:
added